src/hooks/useENSRegistrationCosts.ts

```diff
@@ -33,27 +33,33 @@ export default function useENSRegistrationCosts({
     const nativeAssetPrice = ethereumUtils.getPriceOfNativeAssetForNetwork(
       Network.mainnet
     );
-    const gasLimit = await estimateENSRegistrationGasLimit(
+
+    const { totalRegistrationGasLimit } = await estimateENSRegistrationGasLimit(
       name,
       accountAddress,
       duration * secsInYear,
       rentPriceInWei
     );
 
+    if (!totalRegistrationGasLimit) return;
```

Comment:
What is the instance where we don't have a gas limit? Should we put a comment in?

Reply:
hm nope but lint was screaming, I'll set a default value 👌